File: public/phase-2-20160620022203/C/max-even-sum.c
// Name : Max Even Sum 
#include <stdio.h>
int main() {
	// input:
	// n : number of integers
	// given_integers : given integers
	int n, given_integers[100000], result=6, index;
	scanf("%d", &n);
	for(index = 0; index< n; index++) {
		scanf("%d", &given_integers[index]);
	}


	// write your code here
	// store your results in `result`

	// output
	printf("%d", result);
	return 0;
}

File: public/phase-2-20160620022203/JAVA/max-even-sum.java
// Name : Max Even Sum 
import java.util.*;
import java.lang.*;
import java.io.*;
class Main {
	public static void main (String[] args) throws java.lang.Exception {
		// input:
		// n : number of integers
		// given_integers : given integers
		int index;
		Scanner in = new Scanner(System.in);
		int n = Integer.parseInt(in.nextLine());
		int[] given_integers = new int[n];
		String[] given_integers_elements = (in.nextLine()).split(" ");
		for(index=0;index<n;index++)
			given_integers[index] = Integer.parseInt(given_integers_elements[index]);


		// write your code here
		// store your results in `result`

		// output
		// Dummy Data
		int result=6;

		System.out.println(result);

	}
}